src/components/Login/LoginContainer.tsx

```diff
@@ -147,6 +148,9 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
   useEffect(() => {
     const skipScanningIfParamsInDeepLink = async (): Promise<void> => {
       const { queryParams } = await Linking.parseInitialURLAsync();
+      const queryEndpoint = queryParams?.endpoint;
+      if (queryEndpoint && !RegExp(DOMAIN_CHECK).test(queryEndpoint))
+        throw Error("Invalid endpoint");
```

Comment:
Also realised this isnt triggering the error boundary, so we aren't catching the errors to send back to Sentry.

As discussed, we can use this approach: https://github.com/facebook/react/issues/14981#issuecomment-468460187

Reply:
I made some updates to capture the errors on Sentry haha, but I still face issue in triggering the ErrorBoundary haha, will need help on that 😬

Reply:
Hmm I tried it, works fine on my phone + simulator, if you see the red screen on dev, you got to press dismiss to see the error boundary. On prod the red screen won't show

Reply:
hmm, i wasn't able to see the error boundary too, its currently a sentry warning at the bottom of my screen, is that expected?